compiler/src/iree/compiler/Codegen/Common/TransformDialectStrategiesGPU.cpp

```diff
@@ -551,9 +579,13 @@ static void createReductionStrategyStagedThreadDistribution(
   // Map the potential maybeTiledLeadingH.
   // TODO: Consider fusing leading elementwise into threads.
   if (strategy.captures.maybeLeadingRank > 0) {
+    int64_t vectorSize =
+        (4 * 32) / strategy.captures.maybeLeadingOutputElementalTypeBitWidth;
```

Comment:
Consider getting rid of magic numbers. IMO, `kVectorLoadBitWidth = 128` is more justifiable here than `4 * 32`.

Reply:
Actually, there's even a constant declared in another file, just use it.